app/helpers/fhir_helper.rb

```diff
@@ -36,6 +36,7 @@ def patient_as_fhir(patient)
     FHIR::Patient.new(
       meta: FHIR::Meta.new(lastUpdated: patient.updated_at.strftime('%FT%T%:z')),
       contained: [FHIR::Provenance.new(
+        id: SecureRandom.uuid,
```

Comment:
I actually think it makes more sense to just take this out. I added it because in doing some validation on the resources we produced, I got the impression that a contained resource should have an `id`, but after some further investigation, it actually isn't really necessary, and in this case I think it's probably more confusing than anything, because there is no reason why you would ever want to independently identify these `Provenance` resources. And I don't want to cause confusion between the contained `Provenance`, and our usage of `Provenance` to map to history.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1016/commits/32602e5574a2690b5100fe9319363505e2faf485 should address this